src/pages/qr-menu/index.tsx

```diff
@@ -42,6 +43,15 @@ export default function QrMenuPage() {
           </div>
         </div>
       </Section>
+      <Plans />
+      <Section theme="white">
+        <div className="margin-top-0">
```

Comment:
this can be a blank <> and it will automatically remove the top margin off of the h2 below it. if you check scss for the Section component you can see that all top margins are stripped from the first item and bottom margins stripped from the last item within the section